src/script/run-cbt.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+usage() {
+    local prog_name=$1
+    shift
+    cat <<EOF
+$prog_name [options] <config-file>...
+
+options:
+  -a,--archive-dir    directory in which the test result is stored
+  -h,--help           print this help message
+  --cbt               directory of cbt if you have already a copy of it
```

Comment:
Maybe `--cbt-dir` instead for the sake of being coherent with `--archive-dir`?

Reply:
yeah, i thought about this. and after a second thought just now, i still think that `--cbt` is better. the reason why i have `archive-dir` or `build-dir` is that i try to be consistent with the variable names defined by autoconf and CMake. but `cbt` here is a specific tool. i intent to differentiate it from the directories we store artifacts or test results.